src/assets/translations/messages-en.json

```diff
@@ -61,6 +61,57 @@
       "verify": "Verify"
     }
   },
+  "wProvider": {
+    "keepKey": {
+      "header": "Enter Your Pin",
+      "body": "Use PIN layout shown on your device to find the location to press on the pin pad.",
+      "button": "Unlock"
+    },
+    "shapeShift": {
+      "nImport": {
+        "header": "Import your wallet",
+        "body": "Enter your 12 word seed phrase to import an exisiting wallet",
+        "button": "Next"
+      },
+      "nPasswd": {
+        "header": "Enter your password",
+        "body": "Enter a password to encrypt your wallet. In order to securely store your keys we will encrypt them, choose a string password you can remember.",
+        "button": "Next"
+      },
+      "nPassReq": {
+        "header": "Enter your password",
+        "body": "Enter a password to encrypt your wallet. In order to securely store your keys we will encrypt them, choose a string password you can remember.",
+        "button": "Next"
+      }, 
+      "nSeed": {
+        "header": "Your seed phrase",
+        "body": "Write these 12 words down and store them securely offline. This 12 word phrase is used to recover your wallet private keys.",
+        "button": "Next"
+      }, 
+      "nStart": {
+        "header": "Create or import a wallet",
+        "body": "Would you like to create a new wallet or import an existing one?",
+        "button": "Import Wallet",
+        "button2": "Create Wallet"
+      }, 
+      "nSuccess": {
+        "header": "Wallet Conected",
+        "success": "Your wallet has been connected",
+        "error": "There was an error connecting your wallet"
+      },
+      "nTest": {
+        "header": "Let's practice your secret phrase",
+        "body": "Which of these is the",
+        "body2": " word",
+        "body3": "in your secret phrase.",
+        "button": "Next"
+      }
+    },
+    "sModal": {
```

Comment:
what does `sModal` denote? would this be better named `sendModal`?

Reply:
sModal is an abbreviation for SelectModal.tsx